packages/integration_test/lib/integration_test_driver.dart

```diff
@@ -81,11 +81,14 @@ Future<void> integrationDriver({
 
   await driver.close();
 
+//Below mentioned code to get the response data has been moved out of the if block for pass teste so that
+//json report gets created irrespective of test fail or test pass status.
```

Comment:
This comment won't make sense in ten years.
I would probably just not have the comment. It doesn't really say anything that the code itself doesn't already say.

Reply:
Hi @Hixie 
I am using integration_test package and flutter drive command was not generating any json report. flutter test --machine was not working for integration tests.

I wanted a json report in order to create an HTML report for my test execution. So, I used flutter_gherkin plugin which was fulfilling my requirement. With flutter_gherkin when I was using flutter drive --driver=test_driver/integration_test_driver.dart --target=integration_test/gherkin_suite_test.dart command to execute my tests, json report was not generated when one or more tests were failing.

When I made the changes mentioned in the PR, json was generating for pass as well as failed tests. Thereafter, I used that json report to create an HTML report. Here is the link to my repo where you can see the code - https://github.com/umangdivyanshu/WatchAppPoc/tree/bdd-refactoring